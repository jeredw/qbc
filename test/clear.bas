clear
clear 1
clear 1, 2
clear 1, 2, 3
clear , 1
clear , 1, 2
clear , , 2
clear 1, , 2
