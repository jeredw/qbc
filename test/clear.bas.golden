(program 
  (statement 
    (clear_statement clear)) \n 
  (statement 
    (clear_statement clear 
      (expr 
        (literal 1)))) \n 
  (statement 
    (clear_statement clear 
      (expr 
        (literal 1)) , 
      (expr 
        (literal 2)))) \n 
  (statement 
    (clear_statement clear 
      (expr 
        (literal 1)) , 
      (expr 
        (literal 2)) , 
      (expr 
        (literal 3)))) \n 
  (statement 
    (clear_statement clear , 
      (expr 
        (literal 1)))) \n 
  (statement 
    (clear_statement clear , 
      (expr 
        (literal 1)) , 
      (expr 
        (literal 2)))) \n 
  (statement 
    (clear_statement clear , , 
      (expr 
        (literal 2)))) \n 
  (statement 
    (clear_statement clear 
      (expr 
        (literal 1)) , , 
      (expr 
        (literal 2)))) \n)
